From 63c2c00e9bcf663f93e5b13cd6d147ba83ae9e08 Mon Sep 17 00:00:00 2001
From: Alberto Milone <alberto.milone@canonical.com>
Date: Tue, 6 Aug 2013 14:14:21 +0200
Subject: [PATCH 1/1] Add support for Linux 3.11

nv-linux.h: use totalram_pages if num_physpages is not available.
---
 nv-linux.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/nv-linux.h b/nv-linux.h
index 9aac81a..06aad74 100644
--- a/usr/src/nv/nv-linux.h
+++ b/usr/src/nv/nv-linux.h
@@ -15,6 +15,10 @@
 #include "nv.h"
 #include "conftest.h"
 
+#ifndef num_physpages
+#define num_physpages totalram_pages
+#endif
+
 #if defined(NV_GENERATED_AUTOCONF_H_PRESENT)
 #include <generated/autoconf.h>
 #else
-- 
1.7.9.5

